### PR TITLE
add validation criteria for featurize

### PR DESCRIPTION
**Pull request by chirranjeevigopal-TRI**
Added a switch to initialize dQ features to be np.nan if Qddiff is an empty array. THis occurs for cells discharged rapidly over a narrow voltage window, which do not have `cycles_interpolated` discharge step.

### CODE REVIEW

File: beep/featurize.py [hunk 1]
```diff
@@ -623,24 +624,24 @@ def features_from_processed_cycler_run(cls, processed_cycler_run):
         Vd = interpolated_df.voltage[interpolated_df.cycle_index == iini]
         Qd_diff = Qd_final.values - Qd_10.values
 
-        X[5] = np.log10(np.abs(np.min(Qd_diff)))  # Minimum
-        labels.append("abs_min_discharge_capacity_difference_cycles_2:100")
+        if len(Qd_diff):
```
Comment (patrickherring-TRI) [inline beep/featurize.py:627]: Could we include a comment here just as a reminder about why this logic exists?

File: beep/structure.py [hunk 2]
```diff
@@ -749,7 +749,8 @@ def from_maccor_file(cls, filename, include_eis=True, validate=False):
         return cls(data, metadata, eis, validate, filename=filename)
 
     def determine_structuring_parameters(self, v_range=None, resolution=1000,
-                                         nominal_capacity=1.1, full_fast_charge=0.8):
+                                         nominal_capacity=1.1, full_fast_charge=0.8,
+                                         parameters_path='data-share/raw/parameters'):
```
Comment (patrickherring-TRI) [inline beep/structure.py:753]: Could we also add the argument to the docstring?

File: beep/featurize.py (hunk 1 above)
Comment (chirranjeevigopal-TRI) [reply beep/featurize.py:629]: Done!

File: beep/structure.py (hunk 2 above)
Comment (chirranjeevigopal-TRI) [reply beep/structure.py:753]: Added arg to docstring

**Pull request closed without merging**